components/html-block/README.md

```diff
@@ -0,0 +1,21 @@
+# HTML Block
+
```

Comment:
Nit: I think typically for READMEs that only have one thing in them we don't duplicate headings like this.

Reply:
Thanks.  I will fix that up next time I am in here making changes.